src/components/service.css

```diff
@@ -0,0 +1,78 @@
+@font-face {
+    font-family: 'Inter';
+    src: local('Inter'), url(../fonts/Inter/Inter-VariableFont_slnt,wght.ttf) format('truetype');
+    font-weight: 100 1000;
+	font-stretch: 25% 151%
+  }
+
+div{
+    font-family: Inter;
+    font-weight: 350;
+    text-align: left;
+}
+
+.heading{
+    margin: auto;
+    width: 50%;
+    padding: 10px;
+    text-align: center;
+    top: 100px;
+    color: rgb(255, 255, 255);
+}
+
+.textBackground {
```

Comment:
"Container" rather than "Background" is the usual terminology. Also css styles should be hyphenated & lowercase e.g "text-container"